tools/Azure.Mcp.Tools.KeyVault/tests/Azure.Mcp.Tools.KeyVault.LiveTests/KeyVaultCommandTests.cs

```diff
@@ -6,13 +6,23 @@
 using System.Text.Json;
 using Azure.Mcp.Tests;
 using Azure.Mcp.Tests.Client;
+using Azure.Mcp.Tests.Client.Helpers;
+using Azure.Mcp.Tests.Generated.Models;
 using Azure.Security.KeyVault.Keys;
 using Xunit;
 
 namespace Azure.Mcp.Tools.KeyVault.LiveTests;
 
-public class KeyVaultCommandTests(ITestOutputHelper output) : CommandTestsBase(output)
+public class KeyVaultCommandTests(ITestOutputHelper output, TestProxyFixture fixture) : RecordedCommandTestsBase(output, fixture)
 {
+    public override List<BodyRegexSanitizer> BodyRegexSanitizers => new List<BodyRegexSanitizer>() {
```

Comment:
I don't know if we should proactively sanitize/redact the contents of a certificate response to avoid triggering security alerts. [Here's how](https://github.com/Azure/azure-sdk-for-java/blob/4f7f2e3ebf614dbff43372a086fd5fa0c1095c16/sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java#L106) we did it for Java back in the day when recordings used to live in the same repo as our code. How does it work now that we have a separate assets repo?

Reply:
Yep. This shows up in the recording side as an initial `401` on the first request, and the response has a `www-authenticate` header with the bearer token id. (We sanitize this out  for playback)

Reply:
Once I get `playback` working here, I'll share the recordings with you before I push anything and get your `signoff`.

Reply:
Shared recordings offline for review.